voter-dapp/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "UMA Voter DApp",
+  "name": "UMA Voter DApp",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
```

Comment:
What are these used for? Should they be set to the UMA colors?

Reply:
Removed these colors.